test/test_integration_credentials.py

```diff
@@ -247,13 +247,13 @@ def test_https_logout_successful(self):
                                  ['logout'], exit_code=0)
 
         self.assertEquals(output.strip(), 'Logged out of Conjur')
-        assert os.path.getsize(DEFAULT_NETRC_FILE) == 1
+        with open(DEFAULT_NETRC_FILE) as netrc_file:
+            assert netrc_file.read().strip() == "", 'netrc file is not empty!'
```

Comment:
what the `netrc file is not empty!'` does?

Reply:
in different OS empty files are treated differently. In Windows an empty file as an extra line. so here we are verifying that the file is empty by reading and striping it and if the file isn't empty, we fail and return that message "netrc file is not empty!"